CMakeLists.txt

```diff
@@ -182,6 +182,8 @@ option(BUILD_SQLSMITH "Enable building of SQLSmith." FALSE)
 option(BUILD_TPCE "Enable building of the TPC-E tool." FALSE)
 option(JDBC_DRIVER "Build the DuckDB JDBC driver" FALSE)
 option(BUILD_PYTHON "Build the DuckDB Python extension" FALSE)
+option(USER_SPACE "Build the DuckDB Python in the user space" FALSE)
```

Comment:
This option is defined but doesn't appear to be used.

Reply:
this is used in the src/cmakelists